bluepysnap/schemas/schemas.py

```diff
@@ -2,9 +2,10 @@
 from pathlib import Path
 
 import h5py
+import importlib_resources
 import jsonschema
 import numpy as np
-import pkg_resources
+import referencing
```

Comment:
The library `referencing` should be added explicitly as a requirement in setup.py.
I suppose (but I didn't verify) that we should also pin `jsonschema>=4.18.0`, since it's the version that starts using referencing.